config/locales/en.yml

```diff
@@ -262,7 +264,7 @@ en:
         <p class="govuk-body">If you’re eligible and you said that you do not have a way of getting basic supplies at the moment we will start sending you a weekly box of basic supplies.</p>
         <p class="govuk-body">We’ll also share your details with participating supermarkets to help them prioritise you for their delivery services.</p>
         <p class="govuk-body">If you do not already have an account with a supermarket delivery service, set one up now. If you’re concerned about not being able to get a delivery slot, you can set up accounts with more than one supermarket.</p>
-        <p class="govuk-body">It may take some time before you start getting support. <a class="govuk-link" href="https://www.gov.uk/find-local-council">Contact your local authority</a> if you need supplies urgently and cannot rely on family, friends or neighbours.</p>
+        <p class="govuk-body">This can take up to a week - or longer if you haven't had the letter from your NHS or from your doctor. <a class="govuk-link" href="https://www.gov.uk/find-local-council">Contact your local authority</a> if you need supplies urgently and cannot rely on family, friends or neighbours.</p>
```

Comment:
```suggestion
        <p class="govuk-body">This can take up to a week - or longer if you haven’t had the letter from your NHS or from your doctor. <a class="govuk-link" href="https://www.gov.uk/find-local-council">Contact your local authority</a> if you need supplies urgently and cannot rely on family, friends or neighbours.</p>
```